test/table.test.ts

```diff
@@ -76,7 +76,7 @@ export async function typeCheck() {
   t2 = t3;
 
   t3 = t1;
-  t3 = t2;
+  // t3 = t2; // regression: https://github.com/functionless/functionless/pull/519
```

Comment:
?

Reply:
`Table<Item>` is no longer a sub-type of `Table<Record<string, any>>`.

Would love to fix it but it is not a priority IMO.